PUBLISH.md

```diff
@@ -0,0 +1,42 @@
+# Publish guide
```

Comment:
We should also call out the process to obtain permissions for:
1. npmjs plaid-node package
2. github permissions to push directly to the protected master branch (or re-evaluate this restriction)

Reply:
I'm gonna leave this out of this public md and put it in the private doc